File: matiasmasso/Mat.NET/DTO/DTORebut.cs
using System;

namespace DTO
{
    public class DTORebut
    {
        public DTOLang Lang { get; set; }
        public string Id { get; set; }
        public DTOAmt Amt { get; set; }
        public DateTime Fch { get; set; }
        public DateTime Vto { get; set; }
        public string Concepte { get; set; }
        public string IbanDigits { get; set; }
        public string Nom { get; set; }
        public string Adr { get; set; }
        public string Cit { get; set; }

        public DTORebut(DTOLang oLang = null/* TODO Change to default(_) if this is not a reference type */, string sId = "", DTOAmt oAmt = null/* TODO Change to default(_) if this is not a reference type */, DateTime DtFch = default(DateTime), DateTime DtVto = default(DateTime), string sNom = "", string sAdr = "", string sCit = "", string sConcepte = "", string sIbanDigits = "") : base()
        {
            Lang = oLang;
            Id = sId;
            Amt = oAmt;
            Fch = DtFch;
            Vto = DtVto;
            Concepte = sConcepte;
            IbanDigits = sIbanDigits;
            Nom = sNom;
            Adr = sAdr;
            Cit = sCit;
        }
    }
}

File: matiasmasso/Mat.NET/DT2/DTORebut.vb
Public Class DTORebut
    Property Lang As DTOLang
    Property Id As String
    Property Amt As DTOAmt
    Property Fch As Date
    Property Vto As Date
    Property Concepte As String
    Property IbanDigits As String
    Property Nom As String
    Property Adr As String
    Property Cit As String

    Public Sub New(
        Optional ByVal oLang As DTOLang = Nothing,
        Optional ByVal sId As String = "",
        Optional ByVal oAmt As DTOAmt = Nothing,
        Optional ByVal DtFch As Date = Nothing,
        Optional ByVal DtVto As Date = Nothing,
        Optional ByVal sNom As String = "",
        Optional ByVal sAdr As String = "",
        Optional ByVal sCit As String = "",
        Optional ByVal sConcepte As String = "",
        Optional ByVal sIbanDigits As String = "")

        MyBase.New()
        _Lang = oLang
        _Id = sId
        _Amt = oAmt
        _Fch = DtFch
        _Vto = DtVto
        _Concepte = sConcepte
        _IbanDigits = sIbanDigits
        _Nom = sNom
        _Adr = sAdr
        _Cit = sCit
    End Sub
End Class
